WordPress/Sniffs/WhiteSpace/ControlStructureSpacingSniff.php

```diff
@@ -496,7 +496,8 @@ public function process_token( $stackPtr ) {
 			return;
 		}
 
-		$trailingContent = $this->phpcsFile->findNext( PHP_CodeSniffer_Tokens::$emptyTokens, ( $scopeCloser + 1 ), null, true );
+		// {@internal This is just for the blank line check. Only whitespace should be considered.}}
```

Comment:
Not required, but felt like the appropriate annotation. It's mostly to prevent me from changing it back when doing spot checks whether the appropriate token is used :innocent: